code/game/objects/items/weapons/tools.dm

```diff
@@ -319,6 +319,8 @@
 
 		if(isliving(O))
 			var/mob/living/L = O
+			message_admins("[key_name_admin(user)] set (key_name_admin(L)) on fire")
+			log_game("[key_name(user)] set [key_name(L)] on fire")

```

Comment:
the mob isn't necessarily set on fire. It needs to have flammable fluid on them to be set on fire...